src/viam/app/data_client.py

```diff
@@ -537,8 +540,8 @@ async def tabular_data_capture_upload(
                 SensorData(
                     metadata=(
                         SensorMetadata(
-                            time_requested=datetime_to_timestamp(data_request_times[idx][0]) if data_request_times[idx][0] else None,
```

Comment:
(flyby) this would have caused a runtime error in cases where the (ostensibly optional) list of tuples was None.